performance_regression_nosqlbench.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from sdcm.tester import ClusterTester
+
+LOGGER = logging.getLogger(__name__)
+
+
+class PerformanceRegressionNosqlBenchTest(ClusterTester):
+    #  pylint: disable=useless-super-delegation
+    def __init__(self, *args):
+        super().__init__(*args)
+
+    def test_mixed(self):
+        """
+        Test steps:
+
+        1. Run a write workload as a preparation
```

Comment:
It actually does, but it's baked into the NoSQLBench scenario we're using (currently: `cql-tabular2`) as the rampup phase. I will update the docstring here, since this is reflected in the test case yaml, not in the code here.

Reply:
ok, please do.

Reply:
Fixed (changed the docstring to a more generic one, as we'll aim to provide the details in the yaml test case files).